packit/api.py

```diff
@@ -1097,25 +1097,34 @@ def _handle_sources(
 
         # btw this is really naive: the name could be the same but the hash can be different
         # TODO: we should do something when such situation happens
```

Comment:
We can remove this comment.

And I don't think it's naive. It's just what `fedpkg` itself uses as well.